README.md

```diff
@@ -431,108 +480,38 @@ Which in a JSON representation would resolve to the following table:
 }
 ```
 
-For more on this representation, please refer to [§6.1](#61-canonical-json-collection).
+For more on this representation, please refer to [§7.1](#71-canonical-json-collection).
 
-# 4. Reserved Capabilities
+# 4. Reserved Resources
 
-The following capabilities are REQUIRED to be implemented.
+The following resources are REQUIRED to be implemented.
 
-## 4.1 Superuser
+# 4.1 `ucan:token`
 
-The superuser ability MUST be denoted `*`. This is the maximum ability and may be applied to any resource (it is the "top" ability).
+### 5.2.1 `ucan:token` Scheme
 
-This is useful in several cases, for example:
-
-1. When delegating all resources, like in a [`ucan:my` scheme](#42-owned-resources)
-2. To grant the maximum ability when the current ability semantics may be extended later
-
-## 4.2 Owned Resources
-
-The `ucan:my` URI scheme represents ownership over a resource — typically by parenthood — at decision-time (i.e. the validator's "now"). Resources that are created after the UCAN was created MUST be included. This higher-order scheme describes delegating some or all ambient authority to another DID.
-
-The use case of "pairing" two DIDs by delegating all current and future resources is not uncommon when a user would like to use multiple devices as "root" but does not have access to all of them directly at all times. An everyday use case for this is a user signing into multiple devices, using them both with full rights.
-
-The format for this scheme is as follows:
-
-``` abnf
-ownershipscheme = "ucan:my:" target
-target = "*" / <scheme> / <uri>
-```
-
-### 4.2.1 `ucan:my` Delegation By Parenthood
-
-Capabilities being delegated by parenthood (the "apex" delegation) MUST be prefixed with `my:`. Including this prefix disambiguates that the source of this capability is the current issuer's owned resources.
-
-Without this prefix, the ambiguity in provenance can be exploited by a malicious user in an unrelated proof, forcing the validly delegated capability to appear as though it has an invalid ownership claim. The `ucan:my:` prefix resolved this issue by making introduction of a capability by parenthood explicit.
-
-#### 4.2.1.1 Example
-
-```js
-// By parenthood
-{
-  "with": "ucan:my:email:alice@example.com",
-  "can": "msg/send"
-}
-
-// The same resource, further delegated
-{
-  "with": "email:alice@example.com",
-  "can": "msg/send"
-}
-```
-
-### 4.2.2 `ucan:my` Wildcard
-
-The wildcard `ucan:my:*` resource MUST be taken to mean "everything" (all resources of all types) that are owned by the current DID.
-
-### 4.2.3 Subschemes
-
-A "sub-scheme" MAY be used to delegate some of that scheme controlled by parenthood. For example, `my:dns` delegates access to all DNS records. `my:mailto` selects all owned email addresses controlled by this user.
-
-### 4.2.4 `ucan:as:` Redelegation
```

Comment:
_Controversial,_ but @matheus23 we can move this into an extension spec for now. It's very important to our use case at Fission, but may not be terribly important to others.

I'm also open to putting this back in, but it's safe to pull out here in a subbranch.